movement/watch_faces/settings/set_time_face.c

```diff
@@ -29,32 +29,12 @@
 #define SET_TIME_FACE_NUM_SETTINGS (7)
 const char set_time_face_titles[SET_TIME_FACE_NUM_SETTINGS][3] = {"HR", "M1", "SE", "YR", "MO", "DA", "ZO"};
 
-void set_time_face_setup(movement_settings_t *settings, uint8_t watch_face_index, void ** context_ptr) {
-    (void) settings;
-    (void) watch_face_index;
-    if (*context_ptr == NULL) *context_ptr = malloc(sizeof(uint8_t));
-}
+static bool _quick_ticks_running;
```

Comment:
IMO this variable would be better in a state struct that is stored in the *context ptr like other faces. Would have to migrate the current use of the context ptr for the page number to it, but that seems straightforward

Reply:
My approach was to only store variables that need some persistence to the state struct. The quick ticks running indicator is only needed when the settings face is the active watch face and is no longer needed when resuming it. So just using a static var seems to be the more straightforward strategy to me.